server/config.go

```diff
@@ -70,21 +83,53 @@ func (s *Service) ConfigSection(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	section := httprouter.GetParamFromContext(ctx, "section")
 	var res interface{}
-	switch section {
-	case "auth":
-		res = newAuthConfigResponse(*config)
-	default:
-		Error(w, http.StatusBadRequest, fmt.Sprintf("received unknown section %q", section), s.Logger)
+	res = newLogViewerUIConfigResponse(*config)
```

Comment:
I think I'd prefer to see something like
```go
res := newLogViewerUIConfig(...)
````

Thoughts?

Reply:
cool with us -- again, not our code here. is there any difference between `var res interface{}` and `res :=`?

Reply:
```
res :=
```

should work here. We only needed the `interface{}` in the original config handlers because they were generic.

Reply:
>  is there any difference between var res interface{} and res :=

the difference is that you lose type information, `interface{}` can be any type and so if you ever want to use it as another type you have to do a type assertion. In this specific case it's probably not too big of a deal. More of just a hygiene thing.

In general it's best to do `... := ...` instead of `var ...`.

Reply:
👍